unleashandroidsdk/src/main/java/io/getunleash/android/UnleashConfig.kt

```diff
@@ -44,8 +44,11 @@ data class UnleashConfig(
             "Authorization" to auth,
             "Content-Type" to "application/json",
             "UNLEASH-APPNAME" to appName,
+            "X-UNLEASH-APPNAME" to appName,
             "User-Agent" to appName,
             "UNLEASH-INSTANCEID" to instanceId,
+            "X-UNLEASH-CONNECTION-ID" to instanceId,
```

Comment:
I hope we can deprecate UNLEASH-INSTANCEID header, as this connection id seems to be fulfilling the same purpose. We should open that conversation, but definitely outside of this PR scope